.ci/cico_build_and_deploy_custom_rh-che.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018 Red Hat, Inc.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+
+set -x
+set -e
+set +o nounset
+
+/usr/sbin/setenforce 0
+
+function retrieveActiveToken() {
+  CURL_OUTPUT=$(curl -H "Content-Type: application/json" -X POST -d '{"refresh_token":"'$KEYCLOAK_TOKEN'"}' https://auth.prod-preview.openshift.io:8443/api/token/refresh)
+  ACTIVE_TOKEN=$(echo $CURL_OUTPUT | jq --raw-output ".token | .access_token")
+  if [[ -z "${RH_CHE_AUTOMATION_CHE_PREVIEW_USERNAME}" ]]; then
+    empty_credentials="OSIO username is empty, "
```

Comment:
`empty_credentials` should be renamed to `error_message` IMO. also consider reworking var not set case as done in other ci script (but it is personal preference) - https://github.com/redhat-developer/rh-che/blob/master/.ci/cico_deploy.sh#L54

Reply:
done.